lib/logstash/plugin_mixins/kafka/avro_schema_registry.rb

```diff
@@ -68,6 +86,23 @@ def check_for_schema_registry_connectivity_and_subjects
       if schema_registry_key and !schema_registry_key.empty?
         options[:auth] = {:user => schema_registry_key, :password => schema_registry_secret.value}
       end
+      if schema_registry_ssl_truststore_path and !schema_registry_ssl_truststore_path.empty?
+        options[:ssl] = {} unless options.key?(:ssl)
+        options[:ssl].merge!({
+          :truststore => schema_registry_ssl_truststore_path,
+          :truststore_password => schema_registry_ssl_truststore_password.value,
```

Comment:
`schema_registry_ssl_truststore_password` might be `nil` as it doesn't have a default value OR don't we need to fail to accept empty password?
```suggestion
          :truststore_password => schema_registry_ssl_truststore_password&.value,
```

Reply:
Good point, maybe for all this settings is better to use the same pattern used at https://github.com/logstash-plugins/logstash-integration-kafka/blob/0462434fc6cf2c1de0d9314bbeb0c0840473d223/lib/logstash/plugin_mixins/kafka/common.rb#L23

Reply:
solved by 4914eb4eb3ddde5094fde9d960ebc697be6fad34